mod-source-record-storage-server/src/main/java/org/folio/dao/RecordDaoImpl.java

```diff
@@ -665,10 +683,28 @@ public Future<Integer> calculateGeneration(ReactiveClassicGenericQueryExecutor t
 
   @Override
   public Future<ParsedRecord> updateParsedRecord(Record record, String tenantId) {
```

Comment:
Do we need to follow the same approach for the updateParsedRecords method?

Reply:
I agree that we should do the same. This has been added. Currently fighting with Sonarqube to get the PR in a green state.